src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -234,24 +234,29 @@
      ;;MESSAGE CONTENT
      [react/view
       content]
-     [link-preview/link-preview-wrapper (:links (:content message)) outgoing]]]
+     [link-preview/link-preview-wrapper (:links (:content message)) outgoing false]]]
    ; delivery status
    [react/view (style/delivery-status outgoing)
     [message-delivery-status message]]])
 
+(def image-max-width 260)
+(def image-max-height 192)
+
 (defn message-content-image [{:keys [content outgoing] :as message} {:keys [on-long-press]}]
-  (let [dimensions (reagent/atom [260 260])
-        visible    (reagent/atom false)
-        uri        (:image content)]
+  (let [dimensions (reagent/atom [image-max-width image-max-height])
+        visible (reagent/atom false)
+        uri (:image content)]
     (react/image-get-size
      uri
      (fn [width height]
-       (reset! dimensions [width height])))
+       (when (< width height)
+         (let [k (/ height image-max-height)]
```

Comment:
It'd be good to either move to a separate function, or add a comment explaining what it's doing, as not immediately clear